idact/gui/Window.py

```diff
@@ -0,0 +1,75 @@
+import sys
+from PyQt5.QtWidgets import QApplication, QPushButton, QDialog, QGridLayout, QLabel, QLineEdit
+from idact.detail.jupyter_app.main import main as deploy_notebook
```

Comment:
There's a problem with this import. PyCharm seems to be fine with that because it is not underlined but I get the ModuleNotFoundError.

Reply:
Are you able to start GUI without PyCharm ?

Reply:
When I run it from the console I get an import error.
![image](https://user-images.githubusercontent.com/26610768/58155678-a3f05d00-7c74-11e9-992f-4a5105f14eea.png)

Reply:
But how do you run from the console ?

Reply:
python Window.py